modules/nodeColorManager/pom.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.gephi</groupId>
+        <artifactId>gephi-plugin-parent</artifactId>
+        <version>0.9.0</version>
+    </parent>
+
+    <groupId>edu.buaa.act</groupId>
+    <artifactId>gephi-plugin-node-color-manager</artifactId>
+    <version>1.0.0</version>
+    <packaging>nbm</packaging>
+
+    <name>Node Color Manager</name>
+
+    <dependencies>
+        <!-- Insert dependencies here -->
+        <dependency>
+            <groupId>org.gephi</groupId>
+            <artifactId>tools-api</artifactId>
+            <version>0.9.0</version>
```

Comment:
seems I've got your point: for the same version of gephi, there can only be one rather than multiple versions of a plugin. eg. for gephi 0.9.0, there is only one version of plugin( Node Color Manager ).

Hmm, so I just fix it.